app-games/src/main/java/com/aptoide/android/aptoidegames/LaunchIntentExtensions.kt

```diff
@@ -7,8 +7,17 @@ import com.aptoide.android.aptoidegames.analytics.presentation.withPrevScreen
 // Deep link
 private const val DEEPLINK_KEY = "ag.link"
 
+val Intent?.hasDeepLink get() = this?.extras?.containsKey(DEEPLINK_KEY) ?: false
+
 fun Intent.putDeeplink(deepLink: String): Intent = putExtra(DEEPLINK_KEY, deepLink)
 
+// Ahab
+private const val AHAB_NOTIFICATION = "AHAB_NOTIFICATION"
+
+val Intent?.isAhab: Boolean get() = this?.getBooleanExtra(AHAB_NOTIFICATION, false) == true
+
+fun Intent.putIfAhab(): Intent = putExtra(AHAB_NOTIFICATION, hasDeepLink)
```

Comment:
The name `putIfAhab` is unclear:
1. What it puts if it's Ahab?
2. In the actual code there is no condition to check if it is Ahab and just always sets the value.